ItalianTranslation_PlanetCoaster.xml

```diff
@@ -12346,28 +12346,28 @@
 		<entry key="PopUp_RidePlacement_CheckListHeader" translation="Per aprire qeust'attrazione, devi:"/>
 		<entry key="PopUp_RidePlacement_ConnectEntranceToPath" translation="Collega entrata al sentiero"/>
 		<entry key="PopUp_RidePlacement_PassCheckListHeader" translation="Per avere una corsia prioritaria, devi:"/>
-		<entry key="PopUp_RidePlacement_PlaceEntrance" translation="Piazza entrata"/>
-		<entry key="PopUp_RidePlacement_PlaceExit" translation="Piazza uscita"/>
-		<entry key="PopUp_RidePlacement_PlaceQueueMerge" translation="Piazza uscita prioritaria"/>
-		<entry key="PopUp_RidePlacement_PlaceQueueSplit" translation="Piazza entrata prioritaria"/>
-		<entry key="PopUp_RidePlacement_StationListHeader" translation="Station List"/>
+		<entry key="PopUp_RidePlacement_PlaceEntrance" translation="Posiziona entrata"/>
+		<entry key="PopUp_RidePlacement_PlaceExit" translation="Posiziona uscita"/>
+		<entry key="PopUp_RidePlacement_PlaceQueueMerge" translation="Posiziona uscita prioritaria"/>
+		<entry key="PopUp_RidePlacement_PlaceQueueSplit" translation="Posiziona entrata prioritaria"/>
+		<entry key="PopUp_RidePlacement_StationListHeader" translation="Lista Stazioni"/>
 		<entry key="PopUp_RideRatings" translation="Giudizi Corsa"/>
 		<entry key="PopUp_RideStats" translation="Statistiche Corsa"/>
 		<entry key="PopUp_RideTesting_AverageSpeed" translation="Velocità media"/>
 		<entry key="PopUp_RideTesting_CycleTime" translation="Durata"/>
 		<entry key="PopUp_RideTesting_Duration" translation="Durata"/>
 		<entry key="PopUp_RideTesting_Excitement" translation="Eccitazione"/>
 		<entry key="PopUp_RideTesting_Fear" translation="Paura"/>
-		<entry key="PopUp_RideTesting_Heatmaps" translation="Heatmaps"/>
-		<entry key="PopUp_RideTesting_Heatmaps_Live" translation="Live Data"/>
-		<entry key="PopUp_RideTesting_Heatmaps_Pass" translation="Dati Mappa Calore"/>
-		<entry key="PopUp_RideTesting_Heatmaps_Pass_N" translation="Pass {VALUE:int}"/>
-		<entry key="PopUp_RideTesting_Heatmaps_PreviousRun" translation="Previous Run Data"/>
+		<entry key="PopUp_RideTesting_Heatmaps" translation="Mappe di Calore"/>
+		<entry key="PopUp_RideTesting_Heatmaps_Live" translation="Dati attuali"/>
```

Comment:
Più che "Dati attuali" sono Dati in diretta, io ripenserei pure a questo

Reply:
Ok, dati in diretta ci può stare